File: src/main/java/com/bibireden/data_attributes/api/item/ItemFields.java
package com.bibireden.data_attributes.api.item;

import java.util.UUID;

import net.minecraft.item.Item;
import net.minecraft.util.Identifier;

/**
 * Helper class that exposes {@link Item#BASE_ATTACK_DAMAGE_MODIFIER_ID} and {@link Item#BASE_ATTACK_SPEED_MODIFIER_ID}.
 * 
 * @author CleverNucleus
 *
 */
public final class ItemFields extends Item {
	private ItemFields() { super(new Item.Settings()); }
	
	public static Identifier attackDamageModifierID() { return BASE_ATTACK_DAMAGE_MODIFIER_ID; }
	
	public static Identifier attackSpeedModifierID() { return BASE_ATTACK_SPEED_MODIFIER_ID; }
}

File: src/main/kotlin/com/bibireden/data_attributes/api/item/ItemFields.kt
package com.bibireden.data_attributes.api.item

import net.minecraft.item.Item

class ItemFields : Item(Item.Settings()) {
    companion object {
        fun attackDamageModifierID() = ATTACK_DAMAGE_MODIFIER_ID
        fun attackSpeedModifierID() = ATTACK_SPEED_MODIFIER_ID
    }
}